english/ninja/ninja.md

```diff
@@ -17,3 +17,9 @@ I'm like a wild animal in the corner
 Waiting for the break of dawn
 Trying to get through the night
 Just a man with the will to survive
+
+you realize that Pokemon is great and can still help you with your troubles,
+you find a man at a flea market selling Pokemon Red,
+Blue, and Yellow all for 20 dollars along with a Gameboy. You buy it. 
+
+[But it's not a game, boy](/english/ninja/ninja.md)
```

Comment:
Hey Caroline,

It was intentional but this will be a good chance for me to try changing it on my own machine and automatically updating my pull request to something that casey can merge!

Thanks for the reply :+1: